asaps/models.py

```diff
@@ -29,6 +29,25 @@ def get_record(self, uri):
         record = self.client.get(uri).json()
         return Record(record)
 
+    def get_all_records(self, rec_type, repo_id):
```

Comment:
I think we should move the logic to determine the endpoint into a different function and just have this function take the endpoint and return a list. There are a few reasons for this. 1) We keep this method focused on doing its one job. 2) It has some parity with the method for getting a single record. 3) Returning only a list allows the function return value to be used directly in a loop.

Reply:
Excellent point, I'll fix that